.gitignore

```diff
@@ -45,6 +45,11 @@ coverage.xml
 # Pycharm
```

Comment:
Oups: sorry.
usually we do not keep the git files in the modules. If @pedrobaeza has 
good reason to keep it then keep it.
Sorry for the noise